Doc/library/asyncio-platforms.rst

```diff
@@ -59,15 +59,20 @@ All event loops on Windows do not support the following methods:
 
 :class:`ProactorEventLoop` has the following limitations:
 
-* The :meth:`loop.add_reader` and :meth:`loop.add_writer`
-  methods are not supported.
+* :meth:`loop.add_reader` and :meth:`loop.add_writer` only accept
+  socket handles (for example, pipe file descriptors are not supported).
+  When called, :func:`select.select` is run in an additional thread.
 
 The resolution of the monotonic clock on Windows is usually around 15.6
 milliseconds.  The best resolution is 0.5 milliseconds. The resolution depends on the
```

Comment:
"The resolution of the monotonic clock on Windows is usually around 15.6 milliseconds."

Unrelated note: that's [no longer true in Python 3.13](https://docs.python.org/dev/whatsnew/3.13.html):

> On Windows, monotonic() now uses the QueryPerformanceCounter() clock for a resolution of 1 microsecond, instead of the GetTickCount64() clock which has a resolution of 15.6 milliseconds.